SConstruct

```diff
@@ -1336,6 +1346,8 @@ if doConfigure :
 	c.Finish()
 
 env.Append( LIBS = [
+		"tbb" + env["TBB_LIB_SUFFIX"],
+		"blosc" + env["BLOSC_LIB_SUFFIX"],
```

Comment:
I was trying to figure out why this was necessary, and I think it's because we're now configuring on a clone of `env` rather than `env` itself, right? `CheckLib()` will add the library to `LIBS` if it is found, but now it's doing that to the clone, not `env`. But this is OK, because all the LIBS are required, and we won't get this far if any one of them has failed?

Reply:
Yep, you got it right